customer-wallet/src/styles/inlineStyles/containers/Customer/checkout.js

```diff
@@ -1,18 +0,0 @@
-import { orange, lightBorderOrange, lightOrange, mainTextColor, borderColor, darkOrange } from 'context/palette'
```

Comment:
OK. So here are the inline styles that we ran into earlier. We could potentially port them to whatever way we like to use now.

My gut is it keep using CSSModules if the app really was using it till now, else we are in trouble with learning the js way of doing css.

Again, whats your thoughts guys.

Reply:
The colors and stuff can just be integrated into the Theming functionality of Material-UI. Other thoughts [here](https://github.com/ChluNetwork/chlu-wallet/pull/68#discussion_r190347263)